src/fstests/runner.py

```diff
@@ -21,6 +22,14 @@ def __init__(self, configs, args):
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
 
+    def check_kvm_xfstest_env(self):
+        try:
+            result = subprocess.run(["kvm-xfstests --help"], check=True, shell=True)
```

Comment:
With `shell=True` you should be able to run this without a list:
```suggestion
            result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
```

Reply:
good as well! I started this using call, but when I went to python 3.6 doc I saw the explicitly recommendation to use `run`, and probably I missed this detail about the list won't be more necessary.